src/miner_blocklist.js

```diff
@@ -0,0 +1,16 @@
+export const MINER_BLOCKLIST = {
+  code: [
+    // CoinHive
+    /window.CoinHive=window.CoinHive/,
+    /CoinHive\.CONFIG/,
+    /coin-hive\.com/,
+    /coinhive\.com/,
+    /cnhv\.co/,
```

Comment:
Hm, using domains is too broad. There are lots of anti-coin mining add-ons and adblockers that include blocklists containing those domains, so we can't match on them. Let' focus on patterns inside the coinhive script that we can detect even in minified/obfuscated code.

Reply:
Don't ask me why I included them like that, I didn't mean to 😆  I think I wanted to find a pattern that might not issue false-positives for adblockers but probably failed along the way.